src/olympia/reviewers/templatetags/code_manager.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag
+def code_manager_url(
```

Comment:
Maybe run Black on this file, it will fix the arguments.

Reply:
I'm not sure what you're looking for. Running `black` on this file leaves everything as is, except for the function signature, and then it replaces all the single quotes with double quotes, which I don't think we want. Also, by placing all the function arguments on a single line it breaks a `flake8` rule (the line is too long).

What exactly is the problem you think needs fixing?

Reply:
ha, you don't have Black configured for this project then. Black would put the closing parent on its own line like this:

```
def code_manager_url(
    a, r, g, s
):

Reply:
but I like the update code too, which is:

```
def code_manager_url(a, r, g,
                     s):
```

Reply:
Yeah, I don't have `black` configured because when I tried it awhile back it seemed to want to make too many unwanted changes. Do you have a config that makes it play nicely with addons-server?

Reply:
Yeah. I use this vim config: https://github.com/willdurand/dotfiles/blob/71c0e7c222f624bc5332092a43e169b3a9f0950e/vim/vimrc#L323-L325

Reply:
Thanks. I went and checked my VSCode settings and I already had it configured to do that. I ran a test and found that it made a bunch of changes to parentheses. The resulting format is fine, but it would mean a lot of churn in a large file where only a small amount of code has changed, so I'm not sure I should use it all the time.